src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -83,24 +83,24 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
       onEventUpdated({
         ...shapeToolData,
         selectedEstimate: { iri: snsPredictedIri, value: snsPredictedFailureRate },
-        probability: Number(snsPredictedFailureRate),
+        probability: snsPredictedFailureRate,
```

Comment:
Not sure to understand, there was no need to cast, or now there is a typescript warning?

Reply:
Fixed.